package.json

```diff
@@ -24,7 +24,7 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
+    "cordova-plugin-file": "^3.0.0",
```

Comment:
also, are you sure this even downloads version 3.0.0 of the plugin? my understanding was that it downloads the file-transfer plugin first because it's above, and since the file transfer plugin has a dependency on it, it downloads the most recent file plugin because that's how the file transfer plugin's dependencies are set. I would also set this to `^4.1.1` instead of `^3.0.0` because I believe that's the most recent version that wasn't broken.